modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/info.scala

```diff
@@ -0,0 +1,15 @@
+package org.tessellation.sdk.infrastructure.consensus
+
+import org.tessellation.schema.peer.PeerInfo
+
+import derevo.circe.magnolia._
+import derevo.derive
+
+object info {
+
+  @derive(encoder)
+  case class ConsensusInfo[Key](
+    latest: Option[Key],
```

Comment:
It's not really reusable if you specify `latest` as a property. I'd name it:
```
{
 key: ..
 peers: [..]
}
```.

Also, why it's an `Option[Key]`. Is it possible to return peers without the consensus key they're participate in? 🤔

Reply:
You are right, it shouldn't be possible. I've changed it to return `NotFound` status in such situation.